README.md

```diff
@@ -1,9 +1,60 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+## Setup Instructions
 
+### Prerequisites
+- **Node.js** (v16+) and **npm**
+- **PostgreSQL** installed and running
 
+### Installation
+1. Clone the repo and navigate to the directory:
+   ```bash
+   git clone <your-repo-url>
+   cd <project-directory>
+   ```
+2. Install dependencies:
+   ```bash
+   npm install
+   ```
 
---- 
+### Environment Variables
+Create the following `.env` files in the project root:
+- **`.env.development`**:
+   ```
+   PGDATABASE=nc_news_dev
+   ```
+- **`.env.test`**:
+   ```
+   PGDATABASE=nc_news_test
+   ```
 
-This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
+### Database Setup
+1. Create databases:
+   ```sql
+   CREATE DATABASE nc_news_dev;
+   CREATE DATABASE nc_news_test;
+   ```
```

Comment:
This step is not needed as the below script of `npm run setup-dbs` does this for you